docs/configuration/http_conn_man/http_conn_man.rst

```diff
@@ -71,11 +71,34 @@ add_user_agent
   :ref:`config_http_conn_man_headers_downstream-service-cluster` headers. See the linked
   documentation for more information. Defaults to false.
 
-.. _config_http_conn_man_tracing_enabled:
-
-tracing_enabled
-  *(optional, boolean)* Whether the connection manager emits :ref:`tracing <arch_overview_tracing>`
-  data to the :ref:`configured tracing provider <config_tracing>`. Defaults to false.
+.. _config_http_conn_man_tracing:
+
+tracing
```

Comment:
generally a sub-level json object gets its own section (see other docs). Can you make a new section on this page, put the nested json docs there, and link to it from here (again see other docs)